src/controllers/navigation.php

```diff
@@ -72,4 +73,18 @@ public function take(int $exercise_id)
 
 		include VIEW_DIR . '/take.php';
 	}
+
+	public function editFulfillment(int $exercise_id, int $fulfillment_id)
+	{
+		$exercise = new Exercise($exercise_id);
+		$fulfillment = new Fulfillment($fulfillment_id);
+
+		$fields = $fulfillment->getFields();
+
+		if (!$exercise->isFulfillmentInExercise($fulfillment)) {
+			lost();
```

Comment:
add a return statement because without it it will include the lost view and include VIEW_DIR . '/take.php';